src/stories/components/BasicModal/CategoryItem/ArrowAccordionCategory.tsx

```diff
@@ -28,7 +29,7 @@ const AccordionCategory: React.FC<Props> = ({ style, category }) => {
         <AccordionDetails>
           <ItemsStyle isLight={isLight}>
             {category?.subcategories?.map((category) => (
-              <div key={category.name}>{category.name}</div>
+              <div key={category.name}>{capitalizeWord(category.name)}</div>
```

Comment:
Why not just add the CSS `text-transform: capitalize`? Do we need to add a different behavior?

Reply:
You are right, I can use this aproach